src/app/shared/components/TopBar/index.tsx

```diff
@@ -1,10 +1,25 @@
-import SelectTeam from './SelectTeam'
+'use client'
+
+import { usePathname } from 'next/navigation'
+
+import BreadCrumb from './BreadCrumb'
+
+const TopBar = () => {
+  const pathname = usePathname()
+
+  const pathArray = pathname.split('/').filter(Boolean)
 
-const TopBar = async () => {
   return (
-    <div className="border-b border-slate-500 flex flex-col h-topBarHeight items-center mb-1 w-full">
-      <div className="flex items-start max-w-column w-full p-2">
-        <SelectTeam />
+    <div className="border-b border-slate-500 flex flex-col items-center mb-1 w-full p-2">
+      <div className="flex max-w-column w-full p-2">
+        {!!pathArray.length && pathArray.map((path: string, index: number) => 
+          <BreadCrumb
+            key={path}
+            entityKey={path}
+            isLast={!!pathArray[index+1]}
```

Comment:
this thing works but is misnamed, this condition is true if the element is NOT last

usually isLast logic would be `isLast={pathArray.length === index + 1}`